packages/mui-styled-engine-sc/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/styled-engine-sc",
-  "version": "6.0.0-alpha.29",
+  "version": "6.0.0-beta.0",
```

Comment:
This sounds correct @DiegoAndai?

Reply:
It does, I don't know how this happened though, let me check 😅

Reply:
We were releasing under `6.0.0-alpha` since the master split, weird: https://github.com/mui/material-ui/blame/439c1d5b8265ab85a3505e75654c63e7ddc7a743/packages/mui-styled-engine-sc/package.json

cc: @mnajdova to confirm

Reply:
Yep, we started releasing this under v6 when styled-components v6 was released. It can go into beta too.